circle.yml

```diff
@@ -76,6 +76,25 @@ jobs:
       - restore-gradle-cache
       # --stacktrace is just a temporary flag to help with debugging of ConcurrentModificationException that happens occasionally
       - run: ./gradlew testDebugUnitTest --stacktrace
+      - run:
+          name: Save test results
+          command: |
+            mkdir -p ~/junit/
+            find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} ~/junit/ \;
+          when: always
+      - store_test_results:
+          path: ~/junit
+      - run:
+          name: Save test reports
```

Comment:
reports folder can contain more items then only tests, so we should rename this task to `Save test reports`

![image](https://user-images.githubusercontent.com/530122/81003484-41737b00-8e4b-11ea-92bc-c3a8ebb6482a.png)